tests/test_cluster_utils.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import pytest
+
+from ocsci.testlib import tier1, ManageTest
+from ocs.cluster import CephCluster
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Create disks
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    global cluster
+    cluster = CephCluster()
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    pass
+
+
+@tier1
+@pytest.mark.usefixtures(
+    test_fixture.__name__,
+)
+class TestClusterUtils(ManageTest):
```

Comment:
Please map this to a Polarion valid test

Reply:
@RazTamir ,
This test is for testing CephCluster class and for time being instead of unit test I am having this one.